scripts/zones/East_Ronfaure/TextIDs.lua

```diff
@@ -13,7 +13,7 @@ LOGGING_IS_POSSIBLE_HERE = 7482; -- Logging is possible here if you have
 -- General Dialog
      RAYOCHINDOT_DIALOG = 7406; -- If you are outmatched, run to the city as quickly as you can.
      CROTEILLARD_DIALOG = 7407; -- Sorry, no chatting while I'm on duty.
-      BLESSED_WATERSKIN = 7451; -- To get water, rade the waterskin you hold with the river. 
+      BLESSED_WATERSKIN = 7451; -- To get water, radethe waterskin you hold with the river.
```

Comment:
@takhlaq its prolly special char related, making the T invisible to the batch tools sight